docs/starters.yml

```diff
@@ -6959,3 +6959,25 @@
     - Uses SCSS for easy-to-understand naming
     - Google Analytics support
     - Responsive design
+- url: https://flat-magazine.netlify.app/
+  repo: https://github.com/damnitrahul/gatsby-starter-flat-magazine
+  description: A flat design Gatsby Starter Blog, Powered by Sanity to manage the content and styled using styled-components.
+  tags:
```

Comment:
Refer to [categories.yml](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml) to see the valid tags that can be added here. You cannot have tags like `Styled-components`, `Minimal Design` etc. as they aren't part of `categories.yml` if you believe they are missing you can add request additions to the file. However, we recommend using the available categories.

Reply:
Hey @aaaakshat  thank you for reviewing. I have updated the README and valid `tags`